engine/src/main/java/org/terasology/rendering/opengl/ScreenGrabber.java

```diff
@@ -98,6 +96,11 @@ public void takeScreenshot() {
      * If no screenshot data is available an error is logged and the method returns doing nothing.
      */
     public void saveScreenshot() {
+        // Since ScreenGrabber is initialized before DisplayResolutionDependentFBOs (because the latter contains a reference to the former)
+        // on first call on saveScreenshot() displayResolutionDependentFBOs will be null.
+        if (displayResolutionDependentFBOs == null)
+            displayResolutionDependentFBOs = CoreRegistry.get(DisplayResolutionDependentFBOs.class);
```

Comment:
Noted after merge (missing brackets), @vampcat is aware and will fix come next PR :D